src/libstd/sys/unix/net.rs

```diff
@@ -66,7 +67,30 @@ impl Socket {
     }
 
     pub fn duplicate(&self) -> io::Result<Socket> {
-        let fd = try!(cvt(unsafe { libc::dup(self.0.raw()) }));
+        use libc::funcs::posix88::fcntl::fcntl;
+        const ZERO: c_int = 0;
```

Comment:
How come this and `MINUS_ONE` are constants? Could they just be literals and/or locals?